packages.dhall

```diff
@@ -0,0 +1,128 @@
+{-
```

Comment:
I don't want to add Spago to the core libraries yet. I don't think the value-add of Spago for these libraries is enough to justify having two ways of building them, especially since we should be manually dealing with our dependency version ranges in the core libraries rather than having Spago generate them for us.